.github/workflows/ci.yml

```diff
@@ -32,6 +32,23 @@ jobs:
       - name: Add .github/scripts to $PATH
         run: realpath .github/scripts/ >> $GITHUB_PATH
       - run: make test-pypackage
+  Test_pyapp:
+    name: Test pyapp
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - run: python -m pip install cookiecutter tox
+      - name: Configure git
+        run: |
+          git config --global user.name "GitHub Actions CI Workflow"
+          git config --global user.email "<>"
+      - name: Add .github/scripts to $PATH
+        run: realpath .github/scripts/ >> $GITHUB_PATH
+      - run: make test-pyapp
```

Comment:
This runs the new pyapp cookiecutter's `make sure` on the cookiecutters repo's own CI